### ISSUE TITLE
logorrr.conf should be written after each relevant user interaction

### ISSUE THREAD

**Issue by rladstaetter** [rladstaetter/LogoRRR#248]
Currently, `logorrr.conf` is not always written after a user changes something in the application state. The idea is that the information in `logorrr.conf` should suffice to reinstate the application with the last known state.

For example, if you open a new file, `logorrr.conf` should also be changed. 

It should also be documented somewhere what actions trigger a change in the configuration file.

**Comment by rladstaetter** [comment]
well, if you exit `LogoRRR` and start again, already now logorrr will continue with the same files opened and the same settings used - but depending WHEN `LogoRRR` is exited this is not always correct. 

The selection of filters and their persistence however is really a nuisance atm; works for me, but not necessarily for everybody else ;-)

Here I would like to introduce the concept of `workspaces` where you can save your favourite settings and also choose between different setups. I have to write those ideas down, would be cool if you could give me feedback here :)

**Comment by eitch** [comment]
Right, i was about to write it didn't save, but retried it, and it worked this time. Maybe i closed the file, then exited the app and then it didn't save?

Yes, I'll gladly give feedback on workspaces
